apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -161,7 +161,8 @@ ensure_installed(NameVsn) ->
             ok = purge(NameVsn),
             case ensure_exists_and_installed(NameVsn) of
                 ok ->
-                    maybe_post_op_after_installed(NameVsn);
+                    maybe_post_op_after_installed(NameVsn),
+                    _ = maybe_ensure_plugin_config(NameVsn);
```

Comment:
depends on what `maybe_ensure_plugin_config`, if returns `ok | {error, map()}` then:
```suggestion
                    maybe_ensure_plugin_config(NameVsn);
```

otherwise translate to match `ok | {error, map()}` spec